src/site/content/en/blog/badging-api/index.md

```diff
@@ -158,9 +153,52 @@ Android only ever shows a dot instead of a numeric value.
 Don't assume anything about how the user agent wants to display the badge.
 Some user agents may take a number like "4000" and rewrite it as
 "99+". If you saturate the badge yourself (for example by setting it to "99")
-then the "+" won't appear. No matter the actual number, just set
-`Badge.set(unreadCount)` and let the user agent deal with displaying it
-accordingly.
+then the "+" won't appear. No matter the actual number, just call
+`setExperimentalAppBadge(unreadCount)` and let the user agent deal with
+displaying it accordingly.
+
+While the Badging API *in Chrome* requires an installed app with an icon
+that can be badged, you shouldn't make calls to the Badging API dependent
+on the install state. Just call the API when it exists. If it works,
+it works. If not, it simply doesn't.
+
+## Supporting both the first and second origin trial {: #backwards-compat }
+
+In Chrome 79, the first origin trial ended, and based on developer feedback
+received during the first origin trial, was replaced by a new origin trial
+that uses a slightly different API design. Unfortunately, it is not backwards
```

Comment:
Delete 'slightly'. Renaming the API members and moving them to a different parent may not be a completely different design, but that seems more than slightly different. Also omit needless words.